basyx.aasxfileserver/basyx.aasxfileserver-core/src/test/java/org/eclipse/digitaltwin/basyx/aasxfileserver/core/AasxFileServerSuite.java

```diff
@@ -0,0 +1,155 @@
+	private static final InputStream DUMMY_FILE = new ByteArrayInputStream(byteArray);	
+		
+	@Test
+	public void getAllAASXPackageIds() {
+		
+		AasxFileServer server = getAasxFileServer();		
+		createDummyAASXPackage(server);		
+		Collection<PackageDescription> packageDescriptions = server.getAllAASXPackageIds();			
+		PackageDescription firstPackage = packageDescriptions.iterator().next();	
+
+		assertGetAllAASXPackageIds(packageDescriptions, firstPackage);
+	}
+	
+	@Test
+	public void getAllAASXPackageIdsEmpty() {
+		
+		AasxFileServer server = getAasxFileServer();	
+		Collection<PackageDescription> packageDescriptions = server.getAllAASXPackageIds();					
+				
+		assertIsEmpty(packageDescriptions);		
+	}
+
+	@Test(expected = ElementDoesNotExistException.class)
+	public void getSpecificNonExistingPackageId() {
+		
+		AasxFileServer server = getAasxFileServer();
+		server.getAASXByPackageId("doesNotExist");
+	}
+
+	@Test
+	public void updateExistingAASXByPackageId() {		
+		
+		List<String> expectedAasIds =  new ArrayList<>(Arrays.asList("AAS_ID_3", "AAS_ID_4"));
+		AasxFileServer server = getAasxFileServer();
+		PackageDescription packageDescription = createDummyAASXPackage(server);		
+		server.updateAASXByPackageId(packageDescription.getPackageId(), expectedAasIds, DUMMY_FILE, DUMMY_FILENAME);
+		
+		assertUpdateAASXpaxkageID(packageDescription, expectedAasIds);
+	}	
+
+	@Test(expected = ElementDoesNotExistException.class)
+	public void updateNonExistingAASXByPackageId() {
+		
+		String packageId = "notExisting";		
+		AasxFileServer server = getAasxFileServer();		
+		server.updateAASXByPackageId(packageId, DUMMY_AAS_IDS, DUMMY_FILE, DUMMY_FILENAME);
+	}	
+	
+	@Test
+	public void getAASXByPackageId() throws ElementDoesNotExistException {
+		
+		AasxFileServer server = getAasxFileServer();
+		PackageDescription packageDescription = createDummyAASXPackage(server);			
+		InputStream actualValue = server.getAASXByPackageId(packageDescription.getPackageId());
+		
+		assertEquals(DUMMY_FILE, actualValue);
+	}
+
+	@Test
+	public void deleteAASXPackageById() {
+		
+		AasxFileServer server = getAasxFileServer();
+		PackageDescription packageDescription = createDummyAASXPackage(server);	
+		server.deleteAASXPackageById(packageDescription.getPackageId()); 
+		try {
+			server.getAASXByPackageId(packageDescription.getPackageId());
+			fail();
+		} catch (ElementDoesNotExistException expected) {
+		}
+	}	
+	
+	@Test(expected = ElementDoesNotExistException.class)
+	public void deleteNonExistingAasxFileServer() {
+		
+		AasxFileServer server = getAasxFileServer();
+		server.deleteAASXPackageById("nonExisting");
+	}	
+	
+	private PackageDescription createDummyAASXPackage(AasxFileServer server) {		
+		return server.createAASXPackage(DUMMY_AAS_IDS, DUMMY_FILE, DUMMY_FILENAME);
+	}
+	
+	private void assertGetAllAASXPackageIds(Collection<PackageDescription> packageDescriptions, PackageDescription firstPackage) {		
```

Comment:
Please use expected and actual in the parameters to improve the readability.

Reply:
done